libs/s25main/controls/ctrlBaseImage.h

```diff
@@ -24,8 +24,8 @@ class ctrlBaseImage
     /// Swap the images of those controls
     void SwapImage(ctrlBaseImage& other);
     Rect GetImageRect() const;
-    void DrawImage(const DrawPoint& pos) const;
-    void DrawImage(const DrawPoint& pos, unsigned color) const;
+    void DrawImage(const Rect& dstArea) const;
```

Comment:
Can you add a docstring here what the method is supposed to do when given an area expecially when the area is different than the image size.